File: 1119 - Easy - Remove vowels from a string/solution.kt
class solution {
    fun removeVowels(s: String): String {
        return s.filterNot { 
            it == 'a' || 
            it == 'e' || 
            it == 'i' || 
            it == 'o' || 
            it == 'u'
        }
    }
}

File: 1119 - Easy - Remove vowels from a string/solution.c
char* removeVowels(char * s) {
    int c = 0;
    int size = strlen(s);
    char *ret = malloc(size);
    for (int i = 0; i < size; i++) {
        if (
            s[i] != 'a' &&
            s[i] != 'e' &&
            s[i] != 'i' &&
            s[i] != 'o' &&
            s[i] != 'u'
        ) {
            ret[c] = s[i];
            c++;
        }
    }
    ret[c] = '\0';
    return ret;
}
